.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - '6'
+addons:
+  firefox: "49.0.2"
+before_install:
+  - wget https://github.com/mozilla/geckodriver/releases/download/v0.11.1/geckodriver-v0.11.1-linux64.tar.gz
+  - mkdir geckodriver
+  - tar -xzf geckodriver-v0.11.1-linux64.tar.gz -C geckodriver
+  - export PATH=$PATH:$PWD/geckodriver
+install:
+  - npm install selenium-webdriver@^3.0.0
+  - npm install
```

Comment:
It's already there as part of `npm install`, as axe-core is in the devDependencies.